stacks/spica/src/bucket/pages/index/index.component.ts

```diff
@@ -109,6 +110,7 @@ export class IndexComponent implements OnInit, OnDestroy {
   displayTranslateButton = false;
 
   private postRenderingQueue: Array<any> = [];
+  systemFields: InputPlacerWithMetaPlacer[] = [];
```

Comment:
Sorry, but I couldn't see where we use this property. If we don't use it, can you remove it, please?